frontend/src/scenes/authentication/PasswordReset.tsx

```diff
@@ -55,16 +55,18 @@ function EmailUnavailable(): JSX.Element {
             <LemonDivider className="my-6" />
             <div className="mt-4">
                 If you're an administrator:
-                <ul>
-                    <li>
-                        Password reset is unavailable because email service is not configured.{' '}
-                        <a href="https://posthog.com/docs/self-host/configure/email?utm_medium=in-product&utm_campaign=password-reset">
-                            Read the docs
-                        </a>{' '}
-                        on how to set this up.
-                    </li>
-                    <li>To reset the password manually, run the following command in your instance.</li>
-                </ul>
+                <p>
+                    <ul>
+                        <li>
```

Comment:
Can we put a `<ul>` in a `<p>`? I'm not sure that's semantically valid markup

> The short answer is that ol elements are not legally allowed inside p elements.
-- https://stackoverflow.com/questions/5681481/should-ol-ul-be-inside-p-or-outside

Reply:
?? This is definitely valid and is a commonly used practice